app-start/src/main/java/com/example/android/people/MainActivity.kt

```diff
@@ -112,11 +110,14 @@ class MainActivity : AppCompatActivity(R.layout.main_activity), NavigationContro
     }
 
     override fun openChat(id: Long, prepopulateText: String?) {
-        supportFragmentManager.popBackStack(FRAGMENT_CHAT, FragmentManager.POP_BACK_STACK_INCLUSIVE)
-        supportFragmentManager.commit {
-            addToBackStack(FRAGMENT_CHAT)
-            replace(R.id.container, ChatFragment.newInstance(id, true, prepopulateText))
-        }
+        /* TODO 2 Don't forget about stack
+             add this line
```

Comment:
```suggestion
        /* TODO 2: Setup initial back stack state. In case the user has already opened some chat, remove it.
             add this line
```
@MariaKirdun , давай еще в доке уточним зачем мы эту строчку добавляем